apps/benchmark/tsconfig.json

```
@@ -0,0 +1,17 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "forceConsistentCasingInFileNames": true,
+    "jsx": "react",
+    "outDir": "../../dist/out-tsc",
+    "strict": true,
+    "noImplicitOverride": true,
+    "noPropertyAccessFromIndexSignature": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true
+  },
+  "types": ["environment", "static-assets", "@docusaurus/module-type-aliases", "@docusaurus/theme-classic"],
+  "include": ["./src/**/*.ts", "./src/**/*.tsx", "./src/**/*.js"]
```

```suggestion
  "include": ["./src/**/*.ts", "./src/**/*.tsx", "./*.js"]
```

To include `build.js`

---

Done in [6303a52](https://github.com/microsoft/griffel/pull/132/commits/6303a52d35a5ace09f5d3a05ddf9c4f0ff413b40)